src/Tribe/PUE/Notices.php

```diff
@@ -208,6 +251,17 @@ public function setup_notices() {
 	public function render_invalid_key() {
 		global $pagenow;
 
+		$empty_keys = $this->select_empty_keys();
+
+		// Remove the invalid_key notice for products with an empty license key
+		if ( ! empty( $empty_keys ) ) {
```

Comment:
Invert the logic of the `empty` and return to avoid indentation

Reply:
:zap: Done!